Builds/CMake/CMakeFuncs.cmake

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
-  message(WARNING "In-source builds are strongly discouraged.")
+  message(WARNING "In-root builds are strongly discouraged.")
```

Comment:
Ok, here's a pretty clear definition of "out-of-source" from that same FAQ: https://cmake.org/Wiki/CMake_FAQ#What_is_an_.22out-of-source.22_build.3F, which clarifies "in-source" as its opposite. By that definition, a build under `build/` is pretty clearly "in-source". Given that, since I hate inventing jargon, I'm going to rewrite the messages.